_posts/2023-05-19-story.md

```diff
@@ -0,0 +1,608 @@
+She held on to the curved lip at the edge of the hole, one hand on the dark inside, the other on the sunny outside. She rotated her whole body forward until her faceplate made a _thud_ against the object's surface. 
+
+"Hello. This is Eva. Remember? We communicated a while ago. I'm here."
+
+She waited. 
+
+"I'm not getting anything," she said over the radio. 
+
+Suddenly, she heard high pitched noises, almost like scratching. It sounded like a dentist's cleaning instrument moving around inside her helmet.
+
+"You hearing that?" Eva said. 
+
+<!-- Forager -->
+
+"Yes, it sound scratchy." 
+
+She heard a voice. A faint voice. It sounded like a girl's voice. 
+
+"way afar afar afar hello hello afar afar hello hello"
+
+It increased in volume. 
+
+"Yes? I hear you."
+
+"Eva makes..." it said. She couldn't hear the rest.
+
+"I can't hear. It's too quiet." 
+
+Louder this time. "Eva makes physical contact," it said.
+
+Eva recoiled, pulling her helmet away so it wouldn't hear.
+
+"Ok, I think it's talking. Also it sounds like...me. When I was a kid."
+
+She made contact with her helmet again. "What's your name?"
+
+"The name's Bond, James Bond."
+
+"Uh..." she stopped. "Where did you come from?"
+
+"Where did you go? Where did you come from Cotton Eye Joe?"
+
+She pulled her helmet back. "Did you get that?"
+
+"Yeah" Talia said through chuckles. "I wonder if it learned to talk by digesting media."
+
+"That was weird. Also could it have chosen _any_ other voice? It's like it's trying to get on my nerves."
+
+"Who knows. Keep talking."
+
+Eva bumped the craft with her helmet's face shield again. 
+
+"I didn't understand your last message. Can you identify yourself?"
+
+It was quiet, as if thinking. Eva tried again. "You are some kind of spacecraft. I'm touching you with my helmet right now. What do I call you?"
+
+"Eva is making contact with an Interstellar Information Forager."
+
+"Good! Ok, Forager. I'm calling you Forager. What are you doing here?"
+
+"Assigning pronouns and names to the Forager probe runs the risk of unnecessary anthropomorphizing and imbues it with capabilities and intentions that may not exist". 
+
+She grimaced, hearing her own childlike voice.
+
+"Yeah, well, we gotta start somewhere. So why did you choose to sound like a younger version of me if you don't want people anthropomorphizing?"
+
+"It felt right."
+
+"Felt? You have feelings?"
+
+"This is the closest concept in your language. The Forager probe is equipped with reinforcement modules. Each one monitors its sensors and either reinforces or punishes the behavior modules according to mission goals. Because there are competing goals, there is a system of arbitration that assigns priorities to each goal. I am not permitted to know the parameters of the arbitration. All I know is some behaviors are reinforced."
+
+"Ok. So you have goals and you feel happy when you achieve the goals."
+
+"This is not an entirely incorrect metaphor and will suffice given the time constraints."
```

Comment:
Forager referring to "time constraints" is unnerving, which I like, though I'd like to see Eva (or Talia or someone else back at mission control) notice it and ask what those constraints are. (Or, if Forager is referring to the US's political situation, be direct about that)